components/org.wso2.carbon.identity.auth.abstractotp/src/main/java/org/wso2/carbon/identity/auth/abstractotp/internal/AuthenticatorDataHolder.java

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.auth.abstractotp.internal;
+
+import org.wso2.carbon.identity.event.services.IdentityEventService;
+import org.wso2.carbon.identity.governance.IdentityGovernanceService;
+import org.wso2.carbon.identity.handler.event.account.lock.service.AccountLockService;
+import org.wso2.carbon.idp.mgt.IdpManager;
+import org.wso2.carbon.user.core.service.RealmService;
+
+/**
+ * Encapsulates the data of OTP authenticator.
+ */
+public class AuthenticatorDataHolder {
+
+    private static final AuthenticatorDataHolder authenticatorDataHolder = new AuthenticatorDataHolder();
```

Comment:
Modifiers of the variable says this is a constant. But variable does not follow the constant format.

Reply:
Removed the constant since methods can be invoked through static access